tests/scripts/run_benchmarks.sh

```diff
@@ -20,9 +20,9 @@ touch benchmark_${LASTCOMMIT}.csv
 
 for group in eigen array ; do
     echo "===> Running ${group}.benchmarks ..."
-    ./bin/detray_${group}_masks --benchmark_out=${group}_masks.csv --benchmark_out_format=csv
-    ./bin/detray_${group}_intersect_surfaces --benchmark_out=${group}_intersect_surfaces.csv --benchmark_out_format=csv
-    ./bin/detray_${group}_intersect_all --benchmark_out=${group}_intersect_all.csv --benchmark_out_format=csv
+    ${GITHUB_WORKSPACE}/build/bin/detray_${group}_masks --benchmark_out=${group}_masks.csv --benchmark_out_format=csv
+    ${GITHUB_WORKSPACE}/build/bin/detray_${group}_intersect_surfaces --benchmark_out=${group}_intersect_surfaces.csv --benchmark_out_format=csv
+    ${GITHUB_WORKSPACE}/build/bin/detray_${group}_intersect_all --benchmark_out=${group}_intersect_all.csv --benchmark_out_format=csv
```

Comment:
I worry about this setup a bit. You should set `GITHUB_WORKSPACE` to `PWD` by default, otherwise it will always be a drag to use this script by hand during development.

I have something like the following in mind:

```sh
if [ -z "${GITHUB_WORKSPACE}" ]; then
   BUILD_DIR=${PWD}
else
   BUILD_DIR=${GITHUB_WORKSPACE}/build
fi
```

And then use `${BUILD_DIR}` in these expressions afterwards.

Reply:
Right. `run_benchmark.sh` is now improved so that users can run the analysis locally.